### DISCUSSION TITLE
Meeting of Remix CC (2023-09-24)

### DISCUSSION THREAD

**Discussion by drafish** [PlanckerLabs/remix-cc#77] [Meeting of Remix CC]
### 镜像站
https://remix-project.cn 这个站点要撤掉。remix 团队希望只有一个 remix 官网。ide 镜像站和文档镜像站不受影响，可以继续保留。后面考虑开发一个属于我们自己的官网。

### 教学视频
我觉得教学视频应该是基于用于提出的问题，而不是单纯的介绍功能点。和 Rob 同步了一下我的想法，他很赞同。后面 Remix 团队会收集一些可以作为视频主题的问题，然后给到我们。我们自己也可以在中文社区去收集这些问题。

### 捐赠范围
和 Rob 聊了下下个季度的捐赠范围。我这边建议是以下几个范围
- 中文翻译维护
- IDE 国际化维护
- crowdin 维护
- readthedocs 维护
- 教学视频

目前还没定下来。Rob 说需要和团队讨论下。另外，LearnETH 插件后续有些 UI 更新也可以让我来做，原来复制这部分的是 Pilip ，他有点忙不过来。

**Comment by drafish** [comment]
### DJQ
1. 完成了一个教学视频 -- 在 Remix 上利用 github 来自定义项目模板
2. 和 Remix 开了同步会议
3. 给 yiwen 和 mocca 安排了下任务
4. 星火链网那边已经同意对 remix ide 立项了，下周约个时间再聊下

**Comment by yiwen4** [comment]
### yiwen
1. 写好了cbi的申请
2. 做了一些翻译
3. 准备好了视频的制作，视频正在制作中
